app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  # Include default devise modules.
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable,
+         :omniauthable
```

Comment:
Omniauth enabled but none of the providers are configured. Do you plan to add them on later stage?